frontend/src/components/Auction/AuctionTemplate.tsx

```diff
@@ -42,12 +42,28 @@ interface AutionBidPayload {
     user: string;
 }
 
-const computeMaxBid = ({ bids, price }: Auction) =>
-    bids.reduce((acc, bid) => (bid.value > price ? bid.value : acc), price);
+const computeMaxBid = ({ bids }: Auction) => {
+    if(!bids.length) {
+        return null;
+    } else {
+        const sortedBids = bids.sort((bid1, bid2) => {
+            if(bid1.value > bid2.value) {
+                return -1;
+            } 
+            if(bid1.value < bid2.value) {
+                return 1;
+            }
+            return 0;
+        });
+        return sortedBids[0];
+    }
```

Comment:
You can simplify sorting part to

const sortedBids = bids.sort((bid1, bid2) => bid2.value - bid1.value);